### PR TITLE
Update faq2.md

### PR DESCRIPTION
**Pull request by chris-4444**
Edited the [FAQ subsection](https://wechaty.js.org/docs/explanations/faq) of the Explanations sections.

### CODE REVIEW

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by proudofsimin — APPROVED** [review]

File: docusaurus/docs/explanations/faq.md [hunk 1]
```diff
@@ -4,69 +4,60 @@ title: FAQ
 
 ## Table of Contents
 
-- **General**
-  - [Cannot login](#11-i-can-not-login-with-my-wechat-account)
-  - [What wechaty cannot do on wechat](#a)
+- **General Info**
+  - [Cannot log in using WeChat account](#11-i-can-not-login-with-my-wechat-account)
```
Comment (lijiarui) [inline docusaurus/docs/explanations/faq.md]: It seems the link can not go to the correct place.

File: docusaurus/docs/explanations/faq.md [hunk 2]
```diff
@@ -4,69 +4,60 @@ title: FAQ
...
 
-## General
+## General Info
 
-### I can not login with my Wechat account
+### I cannot log in with my WeChat account
 
-Wechat account that registered after 2017 will not be able to login via Web API. Learn more at [https://github.com/wechaty/wechaty/issues/872](https://github.com/wechaty/wechaty/issues/872)
+- WeChat accounts that are registered after 2017 will not be able to login via Web API. Learn more about it [here](https://github.com/wechaty/wechaty/issues/872).
+- Wechaty also supports protocols other than Web APIs, such as **PadChat**. Learn more about it [here](https://github.com/wechaty/wechaty/issues/1296).
```
Comment (lijiarui) [inline docusaurus/docs/explanations/faq.md]: Please change 
```
- Wechaty also supports protocols other than Web APIs, such as **PadChat**. Learn more about it [here](https://github.com/wechaty/wechaty/issues/1296). 
```

to the following

```
In order to fix this problem, Wechaty also support protocols other than Web APIs, such as:
- [Ipad protocol: PadLocal](http://wechaty.js.org/docs/puppet-services/padlocal)
- [Windows protocol: xp](https://github.com/wechaty/wechaty-puppet-xp)
```

File: docusaurus/docs/explanations/faq.md [hunk 3]
```diff
@@ -4,69 +4,60 @@ title: FAQ
...
+- WeChat accounts that are registered after 2017 will not be able to login via Web API. Learn more about it [here](https://github.com/wechaty/wechaty/issues/872).
+- Wechaty also supports protocols other than Web APIs, such as **PadChat**. Learn more about it [here](https://github.com/wechaty/wechaty/issues/1296). 
 
-Solution: Wechaty support protocols other than Web API, such as pad. Learn more at [https://github.com/wechaty/wechaty/issues/1296](https://github.com/wechaty/wechaty/issues/1296)
+### Does Wechaty support Red Envelope and can transfer money at the moment?
 
-### Does wechaty support Red envelope, transfer money, moment?
+Short answer is **NO** and here are the reasons why:
+- **Payment**: We do not support this because this is related to property security.
+- **@ someone in the room**: We plan to support this in the future and provide solutions other than Web API.
```
Comment (lijiarui) [inline docusaurus/docs/explanations/faq.md]: We already support mention (**@ someone in the room**) in [Ipad protocol: PadLocal](http://wechaty.js.org/docs/puppet-services/padlocal) and [WeCom Solution: WxWork](http://wechaty.js.org/docs/puppet-services/wxwork)

File: docusaurus/docs/explanations/faq.md [hunk 4]
```diff
@@ -4,69 +4,60 @@ title: FAQ
...
+- **Send Contact Card**: We support this in iPad solution.
+- **Send Share Card**: We plan to support this in the future and provide solutions other than Web API.
```
Comment (lijiarui) [inline docusaurus/docs/explanations/faq.md]: We already support Send Share Card in [Ipad protocol: PadLocal](http://wechaty.js.org/docs/puppet-services/padlocal) and [WeCom Solution: WxWork](http://wechaty.js.org/docs/puppet-services/wxwork)

File: docusaurus/docs/explanations/faq.md [hunk 5]
```diff
@@ -4,69 +4,60 @@ title: FAQ
...
+- **Send Voice**: We plan to support this in the future and provide solutions other than Web API.
```
Comment (lijiarui) [inline docusaurus/docs/explanations/faq.md]: We already support Send Voice in [Ipad protocol: PadLocal](http://wechaty.js.org/docs/puppet-services/padlocal)

File: docusaurus/docs/explanations/faq.md [hunk 6]
```diff
@@ -4,69 +4,60 @@ title: FAQ
...
+- **At the moment**: We haven't decided yet whether to support such functions.
```
Comment (lijiarui) [inline docusaurus/docs/explanations/faq.md]: Please change to 

```
- **At the moment**: We are planning to support this, see: https://github.com/wechaty/wechaty/issues/1880
```

File: docusaurus/docs/explanations/faq.md [hunk 7]
```diff
@@ -4,69 +4,60 @@ title: FAQ
...
-- Send Voice: we will support this in the future in solutions other than Web API.
-- Moment: we haven't decide yet whether to support this function
-
-### Can wechaty send url rich media message?
-
-Not yet at this moment, will support later
-
-Related Issue：
+### Can Wechaty send url rich media message?
+Not at the moment but we plan to provide support in the future.
```
Comment (lijiarui) [inline docusaurus/docs/explanations/faq.md]: We already support this in [Ipad protocol: PadLocal](http://wechaty.js.org/docs/puppet-services/padlocal) and [WeCom Solution: WxWork](http://wechaty.js.org/docs/puppet-services/wxwork)

File: docusaurus/docs/explanations/faq.md [hunk 8]
```diff
@@ -4,69 +4,60 @@ title: FAQ
...
+**Related Issues**
 - [Add support for send url rich media message](https://github.com/wechaty/wechaty/issues/718)
-- [can wechaty send share card msg](https://github.com/wechaty/wechaty/issues/824)
-
-### I don't know wechaty support for personal account of wechat official account
+- [Can Wechaty send share card msg?](https://github.com/wechaty/wechaty/issues/824)
 
-At this moment, wechaty only support personal account
+### Does Wechaty provide support for official WeChat personal account?
+Currently, Wechaty only supports personal account created in Wechaty.
```
Comment (lijiarui) [inline docusaurus/docs/explanations/faq.md]: Please change `Currently, Wechaty only supports personal account created in Wechaty.`  to the following:

```
Yes, Wechaty already supports WeChat Official Account, see: 
- GitHub Repo: https://github.com/wechaty/wechaty-puppet-official-account
- Intro: http://wechaty.js.org/docs/puppet-providers/official-account
```

File: docusaurus/docs/explanations/faq.md [hunk 9]
```diff
@@ -4,69 +4,60 @@ title: FAQ
...
+**Related Issue**
+- [Using Wechaty to start a wechatOA account](https://github.com/wechaty/wechaty/issues/1016)
 
 ### What is a `Puppet` in Wechaty
 
-The term `Puppet` in Wechaty is an Abstract Class for implementing protocol plugins. The plugins are the component that helps Wechaty to control the Wechat\(that's the reason we call it puppet\).
+The term `Puppet` in Wechaty is an Abstract Class for implementing protocol plugins. The plugins are the components that help Wechaty to control Wechat\(that's the reason we call it a puppet\).
 
-The plugins are named `XXXPuppet`, like `PuppetPuppeteer` is using the chrome puppeteer to control the WeChat Web API via a chrome browser, `PuppetPadchat` is using the WebSocket protocol to connect with a Protocol Server for controlling the iPad Wechat program.
+The plugins are named `XXXPuppet`; like `PuppetPuppeteer` is using the Chrome puppeteer to control the WeChat Web API via the Chrome browser; `PuppetPadchat` is using the WebSocket protocol to connect with a Protocol Server for controlling the iPad WeChat program.
```
Comment (lijiarui) [inline docusaurus/docs/explanations/faq.md]: PuppetPuppeteer --> PuppetWechat

PuppetPadchat --> PuppetPadLocal

File: docusaurus/docs/explanations/faq.md [hunk 10]
```diff
@@ -4,69 +4,60 @@ title: FAQ
...
 
 ### Wechaty & Queue
 
-In order not blocked by wechat, we add queue in wechaty, see more: [rx-queue](https://github.com/zixia/rx-queue)
-
-### What's the difference between wechaty and wechat4u?
+In order **not** to be blocked by wechat, we add a queue in Wechaty. For more information, refer to [rx-queue](https://github.com/zixia/rx-queue).
```
Comment (lijiarui) [inline docusaurus/docs/explanations/faq.md]: Please change the link to https://github.com/huan/rx-queue

**Review by lijiarui — APPROVED** [review]

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: docusaurus/docs/explanations/faq.md (hunk 2 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/faq.md]: Changed

File: docusaurus/docs/explanations/faq.md (hunk 3 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/faq.md]: Edited bullet point

File: docusaurus/docs/explanations/faq.md (hunk 4 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/faq.md]: Edited bullet point.

File: docusaurus/docs/explanations/faq.md (hunk 5 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/faq.md]: Edited bullet point.

File: docusaurus/docs/explanations/faq.md (hunk 6 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/faq.md]: Edited bullet point

File: docusaurus/docs/explanations/faq.md (hunk 7 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/faq.md]: Edited statement.

File: docusaurus/docs/explanations/faq.md (hunk 8 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/faq.md]: Changed as requested

File: docusaurus/docs/explanations/faq.md (hunk 10 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/faq.md]: Changed link

File: docusaurus/docs/explanations/faq.md (hunk 9 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/faq.md]: Changed names as requested

File: docusaurus/docs/explanations/faq.md (hunk 1 above)
Comment (nibble0101) [reply docusaurus/docs/explanations/faq.md]: Added correct links.

**Review by nibble0101 — APPROVED** [review]
LGTM

**Comment by nibble0101** [comment]
> Please change as I request.

@lijiarui  Can you please have a look at this again?

**Review by shraddhavp — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Close this PR because it has no any activities for months.

**Pull request closed without merging**